docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,99 @@
+Creating and listing backups requires that you:
+* [Have a Charmed MySQL deployed](https://discourse.charmhub.io/t/charmed-mysql-tutorial-managing-units/TODO)
+* [Deploy `cos-lite` bundle in a Kubernetes environment](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s)
```

Comment:
I'd recommend to use the overlay since we need to connect over cross-model relations. I believe it is easiet just to use the `Deploy the COS Lite bundle with overlays` information